src/pages/case-study/central-working-changing-how-you-work.js

```diff
@@ -0,0 +1,385 @@
+import React, { Fragment } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import ReactMarkdown from 'react-markdown'
+import { generate } from 'shortid'
+
+import { Grid, Col } from '../../components/grid'
+import GreyBackground from '../../components/Common/GreyBackground'
+import BlackBackground from '../../components/Common/BlackBackground'
+import TanBackground from '../../components/Common/TanBackground'
+
+import Layout from '../../components/layout'
+import Head from '../../components/Common/Head'
+import Hr from '../../components/Common/Hr'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import { SectionTitle, BodyPrimary } from '../../components/Typography'
+import {
+  TextColumnsBlock,
+  shouldRender,
+  normalise,
+  normaliseAll,
+  BlockRow
+} from '../../components/Common/CaseStudyCards/CaseStudyBlocks'
+
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+import Image from '../../components/Common/Image'
+
+const GreyBodyPrimary = styled(BodyPrimary)`
+  color: ${({ theme }) => theme.colors.secondaryText};
+`
+
+const Block3Col = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[3]};
+`
+
+const Block4ImageCol = styled(Col)`
+  padding-top: ${({ theme }) => theme.space[4]};
+`
+
+const Block6ImageCol = styled(Col)`
```

Comment:
Block4ImageCol & Block6ImageCol are the same. I would use a single component and think of another name

Reply:
I really don't think its necessary to create a shared component here. a `DRY` approach is not applicable in this situation

Reply:
ok. Why? Just trying to understand.